utils/litemap/src/store/vec_impl.rs

```diff
@@ -98,6 +98,39 @@ impl<K, V> StoreMut<K, V> for Vec<(K, V)> {
     }
 }
 
+impl<K: Ord, V> StoreFromIterable<K, V> for Vec<(K, V)> {
+    fn lm_sort_from_iter<I: IntoIterator<Item = (K, V)>>(iter: I) -> Self {
```

Comment:
This is insertion sort, but now that we have a concrete `Vec` type we can `collect`, `sort`, and `dedupe` in `O(n*log(n))`

Reply:
Let's address this in a follow-up